chargeio/services/PaymentMethods/CardOptions.cs

```diff
@@ -1,62 +1,63 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using Newtonsoft.Json;
-
-namespace ChargeIO
-{
-    public class CardOptions : IPaymentMethod
-    {
-        [JsonProperty("type")]
-        public string Type { get { return "card"; } }
-
-        [JsonProperty("card_type")]
-        public string CardType { get; set; }
-
-        [JsonProperty("number")]
-        public string Number { get; set; }
-
-        [JsonProperty("cvv")]
-        public string Cvv { get; set; }
-
-        [JsonProperty("exp_month")]
-        public int? ExpMonth { get; set; }
-
-        [JsonProperty("exp_year")]
-        public int? ExpYear { get; set; }
-
-        [JsonProperty("name")]
-        public string Name { get; set; }
-
-        [JsonProperty("address1")]
-        public string Address1 { get; set; }
-
-        [JsonProperty("address2")]
-        public string Address2 { get; set; }
-
-        [JsonProperty("city")]
-        public string City { get; set; }
-
-        [JsonProperty("state")]
-        public string State { get; set; }
-
-        [JsonProperty("postal_code")]
-        public string PostalCode { get; set; }
-
-        [JsonProperty("country")]
-        public string Country { get; set; }
-
-        [JsonProperty("email")]
-        public string Email { get; set; }
-
-        [JsonProperty("phone")]
-        public string Phone { get; set; }
-
-        [JsonProperty("description")]
-        public string Description { get; set; }
-
-        [JsonProperty("reference")]
-        public string Reference { get; set; }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace chargeio
+{
+    [Serializable]
+    public class CardOptions

```

Comment:
fixed in https://github.com/affinipay/chargeio-net/commit/c9ef65ea76bce4ff9179930ffba1a5f1734c772f